src/Uno.UI/UI/Xaml/UIElement.RoutedEvents.cs

```diff
@@ -506,12 +477,6 @@ public void AddHandler(RoutedEvent routedEvent, object handler, bool handledEven
 			handlers.Add(new RoutedEventHandlerInfo(handler, handledEventsToo));
 
 			AddHandler(routedEvent, handlers.Count, handler, handledEventsToo);
-
-			if (handledEventsToo
-				&& !routedEvent.IsAlwaysBubbled) // This event is always bubbled, no needs to update the flag
```

Comment:
Great your removed the `SubscribedToHandledEventsToo` 👍 But then you should also remove the `IsAlwaysBubbled` flag on the `RoutedEvent`.